src/components/TextInput/BaseTextInput.js

```diff
@@ -285,7 +285,7 @@ class BaseTextInput extends Component {
                                             (!hasLabel || this.props.multiline) && styles.pv0,
                                             this.props.prefixCharacter && StyleUtils.getPaddingLeft(this.state.prefixWidth + styles.pl1.paddingLeft),
                                             this.props.secureTextEntry && styles.secureInput,
-                                            !this.props.multiline && {height: this.state.height},
+                                            !this.props.multiline && {height: this.state.height, lineHeight: undefined},
```

Comment:
I think `lineHeight: undefined` would look a bit odd to anyone coming across it randomly - can we add a comment explaining why it's here and what it's doing?

Reply:
```diff
+ // Explicitly remove `lineHeight` from single line inputs so that long text doesn't disappear once it exceeds the input space (See https://github.com/Expensify/App/issues/13802)
```

How about that?